sencha-workspace/SlateTasksTeacher/app/controller/Dashboard.js

```diff
@@ -293,15 +293,36 @@ Ext.define('SlateTasksTeacher.controller.Dashboard', {
 
         studentTask.set('TaskStatus', status);
 
-        me.doSaveStudentTask(studentTask, function(rec) {
-            taskRater.close();
-            me.getStudentTasksStore().load({
-                id: studentTask.getId(),
-                addRecords: true
-            });
+        me.doSaveStudentTask(studentTask, function(rec, success, request) {
+            var response = JSON.parse(request.getResponse().responseText),
```

Comment:
@nbey see the comment about `JSON.parse` in the below call to `doSaveStudentTask`